teps/README.md

```diff
@@ -0,0 +1,9 @@
+# Tekton Enhancement Proposals (TEPs)
+
+A Tekton Enhancement Proposal (TEP) is a way to propose, communicate
+and coordinate on new efforts for the Tekton project.  You can read
+the full details of the project in
+[TEP-1](0001-tekton-enhancement-proposal-process.md).
+
+This process is still in a _beta_ state and is mandatory for all
```

Comment:
im not quite sure what it means for the process to be in beta state, maybe we can leave that part out?

Reply:
Yeah we can, it was just a way to "protect" the process if it doesn't work :stuck_out_tongue_closed_eyes: